tuxemon/event/actions/add_monster.py

```diff
@@ -75,4 +76,11 @@ def start(self) -> None:
         current_monster.set_level(monster_level)
         current_monster.current_hp = current_monster.hp
 
+        # add moves related to the level
+        current_monster.moves = []
```

Comment:
Should be in set_level (I thought it already was...)

Reply:
No problem, I'll update